lerna.json

```
@@ -0,0 +1,16 @@
+{
+  "npmClient": "yarn",
+  "useWorkspaces": true,
+  "packages": ["packages/create-dao"],
```

```suggestion
  "packages": ["packages/*"],
```

We need this to be a glob for the "Adding more workspaces" instructions to work currently (since there's no instruction to add a new workspace to this array).

It also means those instructions can be more specific by telling contributors to create a new workspace specifically in the `packages` folder.